### ISSUE TITLE
Good recover from real melspectrogram to audio, but problem in autoencoder.

### ISSUE THREAD

**Issue by ericwudayi** [descriptinc/melgan-neurips#23]
Melgan vocoder can convert the real melspectrogram to waveform well, but I find that there is a problem when melgan vocoder convert the "model output melspectrogram".  The melspectrogram extracted from audio is this.
![image](https://user-images.githubusercontent.com/24611279/81052271-bac7a800-8ef5-11ea-92f2-e9bc492540b5.png)

and this one is my autoencoder output:
![image](https://user-images.githubusercontent.com/24611279/81052306-c9ae5a80-8ef5-11ea-8742-9bf20450ce7d.png)

I find that melgan cannot generate good waveform from second one, but the first one is good. (This melspectrogram is not in the training set of my melgan vocoder) Is this the overfitting problem or something else?

**Comment by superhg2012** [comment]
have you solved this problem? I met same issue

**Comment by ericwudayi** [comment]
No, but maybe we can try to solve it. Does it make sense that we feed the melspectrogram output generated by pretrained autoencoder as training data of MelGAN?

**Comment by superhg2012** [comment]
I tried mel-spectrogram generated by Tacotron to train melgan, bad result.  I turn back to ground_truth mel-spectrogram extracted from audio,
